roles/update-access/test/user1.pub

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDURd2gSSwnr41hcF3tlUSx17FJQHBB9a2pFj7qnDkSEYO6r7EVBBUihSXLovwSUqLNz08lJ+4ymd4eM/m9sKkMb0UgsGuTgxpRooIgawcmPOzCi5x8FLjAeI1jmlOpvKdqXK/3bILQUXSFsWXIklso9olBQ0VOPv+RuYVUcPzEhYjSIx8pEg4tmqWU6zDNt4LkYrH7CMwifkWw+Dh6fo+0J7kOotwFxCzqttboVYii9iEazSBNnm6RFSj/6MLsOKdYrqFLiJoVCJK0XSZo+oHHKjucK8G1ZkAOx97g7WDszz9XLj3GSKis+CJpjXnBtgBijV12oYWXs1/jlzCON8hT user1@test.lab
```

Comment:
Can we please name this file id_rsa_user1.pub (or something that indicates that it's the SSH public key)